.github/workflows/regression.yml

```diff
@@ -9,14 +9,20 @@ on:
     branches: 
       - 'develop'
       - 'master'
+  workflow_call:
+    # workflows: ["Regression and Unit Post-Validation on Linux ARM64"]
+    # types: [requested]
+    # branches:
+      # - 'develop'
 
 jobs:
   build:
     name: Build SU2
     strategy:
       fail-fast: false
       matrix: 
-        config_set: [BaseMPI, ReverseMPI, ForwardMPI, BaseNoMPI, ReverseNoMPI, ForwardNoMPI, BaseOMP, ReverseOMP, ForwardOMP]
+        #config_set: [BaseMPI, ReverseMPI, ForwardMPI, BaseNoMPI, ReverseNoMPI, ForwardNoMPI, BaseOMP, ReverseOMP, ForwardOMP]
+        config_set: [BaseMPI]
```

Comment:
Temporarily reduced the config_set to reduce the build time. Once it works I will re-enable the complete list again.